### DISCUSSION TITLE
music/todo/

### DISCUSSION THREAD

**Discussion by giscus** [shane0/music#9] [General]
# music/todo/

https://shanenull.com/music/todo/

**Comment by shane0** [comment]
https://www.ericasynths.lv/shop/standalone-instruments-1/acidbox-iii/

**Reply by shane0** [reply]
# Watch metronome

https://www.soundbrenner.com/pages/pulse

**Comment by shane0** [comment]
https://www.simplifieramp.com/simplifier-manual

**Comment by shane0** [comment]
https://www.nongnu.org/tutka/

**Reply by shane0** [reply]
https://github.com/hexdump0815/linuxsampler-arm-build

**Reply by shane0** [reply]
https://linuxsampler.org/

**Comment by shane0** [comment]
https://www.xlnaudio.com/

**Comment by shane0** [comment]
# next considerations

- https://4mscompany.com/p.php?p=904&c=9

**Reply by shane0** [reply]
strymon star lab

**Comment by shane0** [comment]
# baseline module

- https://youtu.be/6_Qthkmv17o

**Reply by shane0** [reply]
https://www.ericasynths.lv/media/BASSLINE_web.pdf

**Comment by shane0** [comment]
# ua apollo quad dac

| Setting | Line Out | Monitor Out | Recorded to DAW |
|---|---|---|---|
| UAD MON | Processed signal | Processed signal | No |
| UAD REC | Processed signal | Processed signal | Yes |

**Reply by shane0** [reply]
## mac os system sound through ua apollo

**Here's how you can play your internal Mac OS audio through the effects on your UA Apollo:**

1. **Open UA Apollo Console Software:** Launch the software on your Mac.
2. **Select "Console" View:** This will show you the main console interface with channel strips.
3. **Choose a Channel:** Select the channel strip where you want to apply the effects.
4. **Insert Effects:** Click the "Insert Effects" button on the selected channel strip. This will open the effects chain menu.
5. **Choose Effects:** Select the UAD plug-ins you want to use from the menu. You can add multiple effects in series.
6. **Route Internal Audio:**
   * **Option 1: Dedicated Input:** If your UA Apollo has a dedicated input for internal audio (like an "Optical" or "SPDIF" input), connect your Mac's audio output to that input. Then, route that input channel to the channel strip where you've inserted effects.
   * **Option 2: Line Input:** If your UA Apollo doesn't have a dedicated input, you can use a line input. Connect your Mac's audio output to a line input on the Apollo. Then, route that line input channel to the channel strip with your effects.
7. **Enable Monitoring:** Ensure that the "UAD MON" button is lit up yellow on the channel strip. This will allow you to monitor the processed audio in real-time.

**Additional Tips:**

* **Adjust Input Levels:** Adjust the input gain on the channel strip to ensure that the internal audio is not clipping.
* **Experiment with Effects:** Try different UAD plug-ins to achieve the desired sound.
* **Monitor Dry and Wet Signals:** Use the "Solo" buttons on the channel strip to compare the dry (unprocessed) and wet (processed) signals.

By following these steps, you should be able to successfully play your internal Mac OS audio through the effects on your UA Apollo.

**Comment by shane0** [comment]
# tabs

- chaconne
- https://stevenlawguitar.com/wp-content/uploads/2018/04/steven-law-bach-bwv-1004-chaconne-tab-1.pdf

**Comment by shane0** [comment]
# starlab

https://www.strymon.net/faq/how-to-use-karplus-strong-mode-on-starlab/

**Reply by shane0** [reply]
https://www.strymon.net/manuals/StarLab_UserManual_RevA.pdf?srsltid=AfmBOorHRvPM6UTJ9Xiz7IyvfU8FVySW8sVgQJ3LqeOlG0v8ps-sld0r
